### ISSUE TITLE
Is there any chance you could port this to work with Chrome/Edge?

### ISSUE THREAD

**Issue by nixsee** [Mathnerd314/repagination#12]
This addon is amazing and is quite literally the only reason I have Firefox installed... Any chance it could be made available for Chrome? Thanks!

**Comment by Mathnerd314** [comment]
So I tried it out in a branch and got it working: https://github.com/Mathnerd314/repagination/tree/chrome. You can use it by cloning the repo and loading it with the extension developer options. It looks like it stays installed after restarting so I don't see any problems with using it that way.
I don't use Chrome that much, so I don't think I'll upload it to the Chrome store (there's a $5 fee).

**Comment by nixsee** [comment]
Wow, thanks! I'm pretty useless with coding stuff, so I'm not sure if I did it fully correctly, but it seems to work! I forked your repo, and then made a pull request to merge chrome branch into patch-2 since the chrome branch didn't seem to want to download to my github desktop. I then was able to load it in the extension options. I'll test it out for a while and then perhaps pay the $5 to upload it for others to use.
